client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
Ha ha and again HA! You really made my day with this comment, thank you.

You're asking for "healthy discussions" but it's you who wrote completely unrelated comment of 2196 characters just to attack me.
I pointed a problem, explained it and expected a healthy discussion you mentioned, but this is what I got. Again.

Also, you ignored a problem pointed out by @xPaw here https://github.com/thelounge/lounge/pull/759#pullrequestreview-54848305 which I overlooked. 
It's the same thing I was talking about, but on a larger scale. With #1047 you absolutely can not click on the text (or even on the empty space left) because
1) message can change while you click
2) it contain clickable usernames

Thus this may result in unexpected behaviour: instead of expanding block, a whois window can be opened.

This is one more proof that your change is really bad UX-wise and I doubt you can say anything against it. 
So feel free to do the same things you always do: ignore it/say that I know nothing about UX/whine about wasted time/ask for a better conduct and healthy discussion.

But it's not that bad, there's something I agree with!
> I do not have the time, the energy, nor the will to deal with this kind of things.

I have the same thought all the time writing previous comments in hope that you stop ignoring problems I pointed. 

ANYWAY, 
I always open for a constructive discussion if you up for it. Also, it's not like my vision of the problem is absolutely right and I really can change it, if you provide arguments instead of forcing your way.
I do want to make Lounge a better client, that's why I'm here. If you really want this too, then listen to other opinions, admit problems and try to find the way to fix them. 

P.S.
> I do not work for you

What is this? I can't understand. Is it supposed to be the answer of the question you quoted? But I'm one of the users. Are you doing this not for every user, but for chosen ones? If so, what the criteria?